Under The Weather/Navigation/Coordinators/WeatherCoordinator.swift

```diff
@@ -5,64 +5,58 @@
 //  Created by Abel Demoz on 14/10/2023.
 //
 
+import Compass
 import UIKit
 
-final class WeatherCoordinator: Coordinator {
+final class WeatherCoordinator: Coordinator, CitySearchNavigationDelegate {
+    func citySelectionNextTapped() {
+        navigator.dismiss(animated: true)
+    }
 
-    typealias Factory = WeatherViewControllerFactory
-    & CitySearchViewControllerFactory
-    & AboutViewControllerFactory
+    typealias Factory = (WeatherViewControllerFactory
+        & CitySearchCoordinatorFactory
+        & AboutViewControllerFactory)
```

Comment:
Not sure why there's brackets here

Reply:
Brackets make no difference. I was trying them out